lib/absinthe/schema.ex

```diff
@@ -556,6 +552,11 @@ defmodule Absinthe.Schema do
   @doc """
   Converts a schema to an SDL string
 
+  By default it excludes any types not referenced directly or transitively from
+  the root query, subscription, or mutation objects. This prevents objects
+  merely used as field imports from appearing in the SDL. If you wish all
+  types to appear, use the `:include_disconnected` option.
```

Comment:
I hate this option name, not sure what a better one is. `:include_unused` would also work, there's also an option for `include: [:unused]` but I don't know what else could go in that list. `:scalars` or something?

Reply:
Other general options could include whether types should be alpha sorted or not. Right now we preserve the original order which is probably the right default, but I know many would prefer it if field / arg / types were all sorted alphabetically.

Reply:
I would definitely go with a single term on this, so `include_unused` is the best option. As for the `include: [:unused]`, if you don't yet know what else to put there, just use a more specific option for now. Later it will be possible to deprecate it painlessly.

Reply:
I'd suggest we don't even bother - let's just match the same behavior as the Introspection mechanism

Reply:
Rendering un-used types via SDL was an unintended side effect of how I put it together in the first place